packages/elements/src/item/index.ts

```diff
@@ -260,6 +261,18 @@ export class Item extends ControlElement {
     return !!(this.labelEl && (this.labelEl.offsetWidth < this.labelEl.scrollWidth));
   }
 
+  /**
+   * Called after the component is first rendered
+   * @param changedProperties Properties which have changed
+   * @returns {void}
+   */
+  protected firstUpdated (changedProperties: PropertyValues): void {
+    super.firstUpdated(changedProperties);
+    if (this.labelEl) {
+      registerOverflowTooltip(this.labelEl, () => this.textContent, () => this.isTruncated);
```

Comment:
This will not work if label has been added after the element initialised

Reply:
Good point!, updated.